pages/pil/exemptions/index.js

```diff
@@ -13,8 +13,10 @@ module.exports = settings => {
   app.post('/', (req, res, next) => {
 
     if (req.body.exempt === 'Yes') {
+      req.session.skipExemptions = false;
```

Comment:
Just realised this probably wants scoping to a profile id...

Reply:
... or maybe pil id?

Reply:
Profile I think.